cli/domain.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+	ct "github.com/flynn/flynn/controller/types"
+)
+
+func init() {
+	register("domain", runDomainMigration, `
+usage: flynn domain set <domain>
+
+Migrate to a new domain.
+
+Examples:
+
+	$ flynn domain set 127.0.0.1.xip.io
+	Success.
+
+	$ flynn domain set 127.0.0.1.xip.io --ca /path/to/ca.cert
```

Comment:
What about `flynn cluster domain` or something like that? It seems weird to have a top-level command and we can add `flynn cluster upgrade`, etc.